### ISSUE TITLE
Allow to set Beta of Moving Window

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/picongpu#377]
Allow the use to set the beta of the moving window
- range `[0.0:1.0]`
- default value: `1.0`

**Comment by ax3l** [comment]
after several refactorings that is easy to implement by now and really needed.
The only thing that needs to be changed is the addition of a user-factor [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/854318c252561ca5f3a372af5cce58213a43a574/src/picongpu/include/simulationControl/MovingWindow.hpp#L58).

**Comment by BrianMarre** [comment]
@steindev  is this issues still relevant?

**Comment by psychocoderHPC** [comment]
Yes, this is still relevant, we have one point where all is already implemented but we do not expose the beta factor currently.
